release-notes/release-notes57.md

```diff
@@ -38,6 +38,9 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
+### Notifications are now shown when the editor is blocked
```

Comment:
```suggestion
### Changes to notifications and popups when the editor is blocked
```
This is not about notifications, it's about the changes that happen when the progress state is set to true. So I've just reworded it to reflect that.